### PR TITLE
[UXE-5390] feat: development of SSO Management creation screen

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### What is the new behavior introduced by this PR? 
* new form for Open ID and SAML for Identity provider
* new services and tests (create/edit/load)
	
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
UIs:

https://github.com/user-attachments/assets/d642323a-37b7-4664-b344-b12a718e29ec

Tests Coverage:

image
image

### Does it have a link on Figma?
no

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [x] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import istanbul from 'vite-plugin-istanbul'
 
 const getConfig = () => {
   const env = loadEnv('development', process.cwd())
-  const URLStartPrefix = env.VITE_ENVIRONMENT === 'production' ? 'https://' : 'https://stage-'
+  const URLStartPrefix = env.VITE_ENVIRONMENT !== 'production' ? 'https://' : 'https://stage-'
```
Comment (aloisio-m-bastian) [inline vite.config.js]: ajusta aqui, deve ter commitado sem quere

Comment (aloisio-m-bastian) [reply vite.config.js]: foi revertido no push a cima

**Review by aloisio-m-bastian — DISMISSED** [review]

**Review by lucasmendes21 — APPROVED** [review]

File: src/views/IdentityProviders/EditView.vue [hunk 2]
```diff
@@ -0,0 +1,153 @@
+<template>
+  <ContentBlock>
+    <template #heading>
+      <PageHeadingBlock pageTitle="Edit Digital Certificate"></PageHeadingBlock>
```
Comment (HerbertJulio) [inline src/views/IdentityProviders/EditView.vue:4]: O título é realmente Digital Certificate?

**Comment by HerbertJulio** [comment]
Outro ponto a considerar é que foi criado um único form, embora existam duas telas uma para criação e outra para edição. Não seria o caso de ter apenas uma tela, já que o conteúdo parece se repetir?

File: src/views/IdentityProviders/EditView.vue (hunk 2 above)
Comment (peterpaulo-azion) [reply src/views/IdentityProviders/EditView.vue]: Ajustado.

**Review by aloisio-m-bastian — APPROVED** [review]

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**